securesystemslib/rsa_keys.py

```diff
@@ -486,7 +503,7 @@ def verify_rsa_signature(signature, signature_scheme, public_key, data):
       # This is a defensive check check..
       else:  # pragma: no cover
         raise securesystemslib.exceptions.UnsupportedAlgorithmError('Unsupported'
-            ' signature scheme is specified: ' + repr(scheme))
```

Comment:
Where is this 'scheme' coming from? I don't see that you removed or renamed scheme in another line. Or was this done in a previous commit, unrelated to this PR?

Reply:
Exactly, it was a [bug](https://github.com/secure-systems-lab/securesystemslib/pull/262/files#r464378530).